services/brig/test/integration/Index/Create.hs

```diff
@@ -84,7 +86,9 @@ testCreateIndexWhenPresent brigOpts = do
     Left e -> fail $ "Invalid ES URL: " <> show esURL <> "\nerror: " <> show e
     Right esURI -> do
       indexName <- ES.IndexName . Text.pack <$> replicateM 20 (Random.randomRIO ('a', 'z'))
-      ES.withBH HTTP.defaultManagerSettings (ES.Server esURL) $ do
+      mgr <- liftIO $ HTTP.newManager HTTP.defaultManagerSettings
+      let bEnv = (ES.mkBHEnv (ES.Server esURL) mgr) {ES.bhRequestHook = ES.basicAuthHook (ES.EsUsername "elastic") (ES.EsPassword "changeme")}
```

Comment:
This block exists three times. We could consider to extract a function to reduce duplication.